generators/server/index.js

```diff
@@ -51,14 +51,7 @@ module.exports = class extends ServerGenerator {
                 // override the displayLogo method from the _initializing phase of JHipster
             },
             mySetupServerconsts() {
-                this.packageName = 'com.example.node';
-                this.buildTool = 'maven';
-                this.enableHibernateCache = false;
-                this.enableSwaggerCodegen = false;
-                this.databaseType = 'mongodb';
-                this.devDatabaseType = 'mongodb';
-                this.prodDatabaseType = 'mongodb';
-                this.skipUserManagement = false;
+                this.packageName = this.configOptions.baseName;
```

Comment:
Why you have removed all configurations?

Reply:
Since we are starting from scratch, I think not having configurations is better. Having them now and letting the main generator catch them makes it generate .java file

Reply:
It's totally wrong. I have defined default values because without them jhipster generation fails (some parameters values are mandatory for jhipster)